File: js/stockPicker.js
exports.picker = function(prices) {
    let biggestProfit = 0
    let result = []
    for(let i = 0; i < prices.length; i++){
        let buyDate = i
        for(let j = i + 1; j < prices.length; j++){
            let sellDate = j
            if(prices[j] - prices[i] > biggestProfit){
                biggestProfit = prices[j] - prices[i]
                result = [buyDate, sellDate]
            }
        }
    }
    return result
}

File: python/stock_picker.py
def picker(prices):
    biggest_profit = 0
    result = []
    for i in range(len(prices)):
        buy_date = i
        for j in range(i + 1, len(prices)):
            sell_date = j
            if prices[j] - prices[i] > biggest_profit:
                biggest_profit = prices[j] - prices[i]
                result = [buy_date, sell_date]
    return result